src/common/todos/reducer.js

```diff
@@ -30,10 +30,10 @@ export default function todosReducer(state = initialState, action) {
 
     case actions.ADD_TODO: {
       const {todo} = action.payload;
-      const newTodo = todo.merge({
+      const newTodo = {...todo,
```

Comment:
Ok, problem is that when you use `persistState` which is awsome, this saves state to JSON and saves it to local storage, so we are losing information about `Record` and we have only object :(

Do you have any idea what to do with this? I thought that you are supposed to send pure object (which will not change when you do `JSON.parse(JSON.strinigy(action.payload))`) to action as payload.